canonical/canonical.com
# copy update: /data/spark/support

## Done

- Fix incorrect copy in "Managed services" section
- Drive by:
  - Fixed indentation
  - Removed stale `canonical_url` jinja block

## QA

- View the site locally in your web browser at: https://canonical-com-1421.demos.haus/data/spark/support
- See that the text under "Go further" and "Backed by an SLA" matches the [doc](https://docs.google.com/document/d/1EcgdnMdW5jfZLLhcisUNMMtsPT-4alaPm3zirtvXEMk/edit?disco=AAABU_UlafU) 

## Issue / Card

Fixes #
